### ISSUE TITLE
Steam

### ISSUE THREAD

**Issue by aceauses** [KrisKamweru/bbs-auto-bot#3]
Greetings, I download the files and install the requirements.. but i try to do this with steam game.. but findTest.py doesnt recognize the steam game.. and always saying "Not Found"..
I want to ask if you gonna make a update about this i really want this story auto bot :)

**Comment by aceauses** [comment]
![image](https://user-images.githubusercontent.com/28159355/113867431-35a8db00-97b7-11eb-9ba1-a230db676620.png)
and also is giving me this error every time i close farmbot.py

**Comment by aceauses** [comment]
![image](https://user-images.githubusercontent.com/28159355/113867504-4d805f00-97b7-11eb-85ac-dd5bb5ca3a68.png)
and this for the findTest.py

**Comment by KrisKamweru** [comment]
Hello, and apologies for the hiatus. I stopped playing BBS some time back and as such didn't really continue developing this bot

However, a few things
This bot was designed for emulator use. I'm not entirely sure it will work on the steam version as I don't exactly know how it takes inputs. It's something I'd have to explore. That is probably why you're not getting it to work. Another possible reason is since updates tend to change the way things look, the bot isn't finding what it looks for because maybe the look of a button changed or something. For now I'd have to wait until I'm back in the game to update that

About the errors you're getting, keyboard interrupt simply means you've forcefully terminated the script. It's not an error with the script itself.

Finally
Me continuing development on this depends a great deal on me coming back to the game. I might do so just to get the script to some level of completion, but I make no promises. However, as this is open source, it's entirely possible for someone to pick up where I've dropped off in the meanwhile.

If I do end up continuing development, some of the things I wanted to add are:
- A GUI so that you just have an easy application to run instead of mucking about with the scripts
- All single player farming run automation (Think single player crystal runs, coin runs and IZ) that can be set up and left to run to the preference of the user
- Some other Quality of Life updates

But, again, no promises unfortunately. I'm not sure if I'll be returning to BBS
